### ISSUE TITLE
Missing version information in Install Dependencies

### ISSUE THREAD

**Issue by langpingxue** [aws-samples/fine-tune-embedding-models-on-sagemaker#1]
Missing version information in the dependencies. 

```
> !pip install datasets
> !pip install transformers
> !pip install transformers[torch]
> !pip install sentence_transformers
```
Follow the content in the FineTuneEmbedding.ipynb, will get an error in part three(Load Data and Train the Model).
The errors seems introduced by the latest changes in some libraries.

**Comment by ennio1991** [comment]
@langpingxue thanks for the feedback. I am working on fixing it, ca you please try the version in the branch [fix/fix-dependency-version](https://github.com/aws-samples/fine-tune-embedding-models-on-sagemaker/blob/fix/fix-dependencis-version/sentence-transformer/multiple-negatives-ranking-loss/FineTuneEmbedding.ipynb) ?

**Comment by langpingxue** [comment]
It works, thank you @ennio1991
